inst/extdata/minimal_scenario2.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Looks good, nice to see multiple trips for multiple folk!

Reply:
Hopefully it works, look forward to see it battle tested, not least in the nascent `activity.Rmd` long form documentation article that would benefit from your :eyes: : )